README.md

```diff
@@ -1,5 +1,54 @@
 # Ktlint Gradle Plugin
 
+This plugin is mainly designed to run [Ktlint](https://github.com/pinterest/ktlint) for changed `kt` files.
+
+## Usage
+
+Run any gradle task from this plugin on CI or locally.
+
+### Tasks
+
+- `ktlintCheck` - Checks code styles for all kotlin files in the project.
+- `ktlintFormat` - Formats all kotlin files in the project.
+- `ktlintCheckUncommittedChanges` - Checks code styles for all changed kotlin files.
+- `ktlintFormatUncommittedChanges` - Formats code style for all changed kotlin files.
+- `ktlintCheckChangesFrom*` - Checks code style for all changed `kt` files against main branch.
+- `ktlintFormatChangesFrom*` - Formats code styles for all changed `kt` files against main branch.
```

Comment:
The `*` at the end is a wildcard for any branch name, right? If it is, then we should update the description. If it's not, then we should remove the wildcard from the task name.